roles/compute_resources/README.md

```diff
@@ -0,0 +1,44 @@
+theforeman.foreman.compute_resources
+====================================
+
+This role creates and manages Compute Resources.
+
+Role Variables
+--------------
+
+This role supports the [Common Role Variables](https://github.com/theforeman/foreman-ansible-modules/blob/develop/README.md#common-role-variables).
+
+The main data structure for this role is the list of `foreman_compute_resources`. Each `compute_resource` requires the following fields:
+
+- `name`: The name of the compute resource.
+
+The following fields are optional and will be omitted by default:
+
+- `description`: Description of the compute resource
+- `provider`: Compute resource provider. Required if *state=present_with_defaults*.
+- `provider_params`: Parameter specific to compute resource provider. Required if *state=present_with_defaults*.
+- `updated_name`: New compute resource name. Setting this will make the role not idempotent.
```

Comment:
Not sure this is really helpful in a role. Maybe renaming is more for ad-hoc ansible calls.